.github/workflows/tests.yml

```diff
@@ -46,15 +46,15 @@ jobs:
           -
         include:
           - os: ubuntu
-            python: 'pypy-3.9'
+            python: 'pypy-3.10'
```

Comment:
This does more than upgrading the github actions

Reply:
Yes.  As mentioned in the commit message.

Reply:
The commit message is literally "Upgrade GitHub Actions"